src/Components/Component1/ErrorBoundary.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function ErrorBoundary(props){
```

Comment:
This ErrorBoundary saver does not work. Please, look into [documentation](https://reactjs.org/docs/error-boundaries.html) to resolve the issue. To check this component use MovieListing variable as a string in the AppComponent component.